backend/app/services/rag/storage/elasticsearch_backend.py

```
@@ -597,14 +633,86 @@ def get_all_chunks(
+                    )
+                    for hit in sample_response.get("hits", {}).get("hits", []):
+                        metadata = (hit.get("_source") or {}).get("metadata") or {}
+                        sample_docs.append(
+                            {
+                                "knowledge_id": metadata.get("knowledge_id"),
+                                "doc_ref": metadata.get("doc_ref"),
+                                "chunk_index": metadata.get("chunk_index"),
+                                "source_file": metadata.get("source_file"),
+                            }
+                        )
+                except Exception as sample_error:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)